aten/src/ATen/native/mps/operations/Convolution.mm

```diff
@@ -66,6 +66,7 @@ Tensor _mps_convolution_impl(
     int64_t groups,
     c10::optional<IntArrayRef> input_shape) {
   TORCH_CHECK(input_t.dim() < 5, "Conv3D is not supported on MPS");
+  TORCH_CHECK(isFloatingType(input_t.scalar_type()), "Convolution is supported only for Floating types");
```

Comment:
I think we support only FP32

Reply:
No, we do support fp16 and we will be supporting bfloat..

Reply:
my bad, FP16 is also supported